### DISCUSSION TITLE
Mixed Reality of the National Election Debates.[Chen Xinyi]

### DISCUSSION THREAD

**Discussion by maotou0913** [realitydeslab/playshop-summer-2024#15] [Proposal]
**Chinese Title**: 美国选举辩论的混合现实
**Your Team**: Chen Xinyi
**Research Questions**:
What will happen to the excessive development of AR in the future where civic participation is prevalent?

**Keywords**:
The US presidential election system, political entertain- ment, augmented reality, speculative design, and partici- pation in democracy

**Artefact / Media**:
Video produced by ai/video produced by myself 
**Abstract**:
Throughout past US presidential elections, a clear trend has been the increasing frequency of personal attacks, on- line entertainment based elections, and the proliferation of false information during the process. And AR glasses technology is proposing more possibilities for informa- tion enhancement and real-time interaction. The “Speech Show” AR application reflects the current ecosystem of the US presidential election, reflecting the relationships between interest groups, the American people, and the in- ternational community. This work provides a series of de- sign speculations and thought-provoking research ideas, which will promote discussions on potential pitfalls and benefits in the political election system.

**Your Reading List**
1.[Anthony Dunne, Fiona Raby - Speculative Everything_ Design, Fiction, And Social Dreaming-The MIT Press (2013).pdf](https://github.com/user-attachments/files/16094880/Anthony.Dunne.Fiona.Raby.-.Speculative.Everything_.Design.Fiction.And.Social.Dreaming-The.MIT.Press.2013.pdf)
2.[The Craft of Research, Fourth Edition (Booth, Wayne C.  Colomb, Gregory G.  Williams etc.) (Z-Library).pdf](https://github.com/user-attachments/files/16094890/The.Craft.of.Research.Fourth.Edition.Booth.Wayne.C.Colomb.Gregory.G.Williams.etc.Z-Library.pdf)

**Visual Reference / Illustrations**
![WechatIMG3107](https://github.com/realitydeslab/playshop-summer-2024/assets/173217378/a1f084c6-b5d8-4352-9888-9042c0d9d585)

**References**:
(1)	Weinmann, C.; Vorderer, P. A Normative Perspective for Political Entertainment Research: Connecting Deliberative Democracy and Entertainment Theory. Commun. Theory 2018, 28 (4), 466–486. https://doi.org/10.1093/ct/qty018.
(2)	Alashri, S.; Kandala, S. S.; Bajaj, V.; Ravi, R.; Smith, K. L.; Desouza, K. C. An Analysis of Sentiments on Facebook during the 2016 U.S. Presidential Election. In 2016 IEEE/ACM International Conference on Advances in Social Networks Analysis and Mining (ASONAM); IEEE: San Francisco, CA, USA, 2016; pp 795–802. https://doi.org/10.1109/ASONAM.2016.7752329.
(3)	Şemsioğlu, S.; Karaturhan, P.; Yantaç, A. E. EMOTE: An Interactive Online Tool for Designing Real-Time Emotional AR Visualizations. In 13th Augmented Human International Conference; ACM: Winnipeg MB Canada, 2022; pp 1–8. https://doi.org/10.1145/3532525.3532527.
(4)	Weinmann, C. Feeling Political Interest While Being Entertained? Explaining the Emotional Experience of Interest in Politics in the Context of Political Entertainment Programs. Psychol. Pop. Media Cult. 2017, 6 (2), 123–141. https://doi.org/10.1037/ppm0000091.
(5)	Chen, J.; Yao, Z.; Zhao, S.; Zhang, Y. Fusion Pre-Trained Emoji Feature Enhancement for Sentiment Analysis. ACM Trans. Asian Low-Resour. Lang. Inf. Process. 2023, 22 (4), 1–14. https://doi.org/10.1145/3578582.
(6)	Qorib, M.; Gizaw, R. S.; Kim, J. Impact of Sentiment Analysis for the 2020 U.S. Presidential Election on Social Media Data. In Proceedings of the 2023 8th International Conference on Machine Learning Technologies; ACM: Stockholm Sweden, 2023; pp 28–34. https://doi.org/10.1145/3589883.3589888.
(7)	Kim, Y. M.; Vishak, J. Just Laugh! You Dont Need to Remember: The Effects of Entertainment Media on Political Information Acquisition and Information Processing in Political Judgment. J. Commun. 2008, 58 (2), 338–360. https://doi.org/10.1111/j.1460-2466.2008.00388.x.
(8)	Filho, W. P.; Rosseti, I.; Viterbo, J. On Tweets, Retweets, Hashtags and User Profiles in the 2016 American Presidential Election Scene. In Proceedings of the 18th Annual International Conference on Digital Government Research; ACM: Staten Island NY USA, 2017; pp 120–128. https://doi.org/10.1145/3085228.3085230.
(9)	Hughes, C. J.; Armstrong, M.; Jones, R.; Crabb, M. Responsive Design for Personalised Subtitles. In Proceedings of the 12th International Web for All Conference; ACM: Florence Italy, 2015; pp 1–4. https://doi.org/10.1145/2745555.2746650.
(10)	Matthes, J.; Heiss, R.; Van Scharrel, H. The Distraction Effect. Political and Entertainment-Oriented Content on Social Media, Political Participation, Interest, and Knowledge. Comput. Hum. Behav. 2023, 142, 107644. https://doi.org/10.1016/j.chb.2022.107644.
(11)	Xia, E.; Yue, H.; Liu, H. Tweet Sentiment Analysis of the 2020 U.S. Presidential Election. In Companion Proceedings of the Web Conference 2021; ACM: Ljubljana Slovenia, 2021; pp 367–371. https://doi.org/10.1145/3442442.3452322.
(12)	Scharl, A.; Weichselbraun, A. Web Coverage of the 2004 US Presidential Election. In Proceedings of the 2nd International Workshop on Web as Corpus - WAC ’06; Association for Computational Linguistics: Trento, Italy, 2006; pp 35–42. https://doi.org/10.3115/1628297.1628303.
(13)	Landreville, K. D.; LaMarre, H. L. Working Through Political Entertainment: How Negative Emotion and Narrative Engagement Encourage Political Discussion Intent in Young Americans. Commun. Q. 2011, 59 (2), 200–220. https://doi.org/10.1080/01463373.2011.563441.

[Mixed Reality of the National Election Debates.pdf](https://github.com/user-attachments/files/16091750/Mixed.Reality.of.the.National.Election.Debates.pdf)

**Comment by maotou0913** [comment]
![Slide 16_9 - 6](https://github.com/realitydeslab/playshop-summer-2024/assets/173217378/73a89fe3-2b59-44ce-a095-daebded43075)

**Comment by maotou0913** [comment]
![Uploading Slide 16_9 - 3.png…]()

**Comment by wuzhuo11** [comment]
https://www.bilibili.com/video/BV1hM411f7Jr/?spm_id_from=333.337.search-card.all.click&vd_source=1c624e4e73618f0c0a1ca007c5670cdf
王世坚送柯市长礼物

**Comment by glitch-kraft** [comment]
@maotou0913 

https://www.instagram.com/reel/C8vxYL4M1al/?igsh=dDZhMm10aG93azli

**Comment by maotou0913** [comment]
Look at how foolish these candidates are, let's download the "Speech Show" app to add more special effects and make them look even more foolish！

**Reply by glitch-kraft** [reply]
**Ttile**
Mixed Reality of the National Election Debates.   

**Tagline**
Watching national election debates in XR will boost your dopamine through expressive effects and graphics. But who financed the development of the app? And how to unmix this mixed reality?
